Umweltzone/src/main/res/values-es/strings.xml

```diff
@@ -0,0 +1,375 @@
+
+
+    <!-- AboutActivity / License -->
+    <string name="appinfo_license_title">Licencias</string>
+    <string name="appinfo_license_content">
+        El código fuente de la aplicación Umweltzone está registrado bajo la licencia GPLv3.
+        Las coordenadas geométricas fueron tomadas de OpenStreetMap y están registradas bajo la Licencia
+        de Bases De Datos Libres (ODbL) v1.0.
+        Otras coordenadas geométricas tienen licencia según lo definido por las fuentes de datos mencionadas.
+        Imágenes y textos están registrados bajo la Licencia Creative Commons Attribution 3.0 Unported.
+    </string>
+    <string name="appinfo_license_url_title_gpl">Licencia GPLv3 </string>
+
+    <!-- AboutActivity / Build Info -->
+    <string name="appinfo_build_info_title">Información de compilación</string>
+    <string name="appinfo_build_info_time">Fecha y hora de compilación</string>
+    <string name="appinfo_build_info_version_code">Número de versión</string>
+    <string name="appinfo_build_info_hash">Hash de compilación</string>
+
+
+    <!-- FeedbackActivity / Contact -->
+    <string name="appinfo_contact_title">Contacto</string>
+    <string name="appinfo_contact_content">
+        Todo tipo de comentarios sobre ideas de como mejorar, reportes de errores
+        o solicitudes de nuevas funcionalidades pueden ser enviadas a la siguiente dirección:
+    </string>
+
+
+    <!-- FeedbackActivity / Rating -->
+    <string name="appinfo_rating_title">Clasificación</string>
+    <string name="appinfo_rating_content">
+        Estamos felices de tener una clasificación positiva en la PlayStore de Google.
+    </string>
+    <string name="appinfo_rating_url_title">
+        Umweltzone en la PlayStore de Google.
+    </string>
+
+    <!-- DetailsActivity formerly CityInfoFragment -->
+    <string name="city_info_activity_title">Información de la ciudad</string>
+
+    <string name="city_info_section_title_lez">Detalles sobre las zonas de baja emisión</string>
+    <string name="city_info_section_title_dpz">Detalles sobre las zonas de prohibición de diésel</string>
+    <string name="city_info_section_title_other">Further information</string>
+
+    <string name="city_info_zone_status">Libre</string>
+    <string name="city_info_zone_number_as_of">
+        A partir del <xliff:g example="1 de Oct del 2099" id="date">%1$s</xliff:g> sólo los
+        vehiculos con etiqueta de color <xliff:g example="verde" id="with_colors">%2$s</xliff:g>
+        pueden ingresar a la zona de bajas emisiones.
+    </string>
+    <string name="city_info_zone_number_since">
+        Desde el <xliff:g example="1 de Feb del 2016" id="date">%1$s</xliff:g> sólo los
+        vehiculos con etiqueta de color <xliff:g example="verde" id="with_colors">%2$s</xliff:g>
+        pueden ingresar a la zona de bajas emisiones.
+    </string>
+    <string name="city_info_zone_number_since_date_format">d &#8217;de&#8217; MMMM &#8217;del&#8217; yyyy</string>
```

Comment:
This crashes the app because this must be a formatting string which defines the country-specific date formatting. Legal characters are `d` for day, `M` for month and `Y` for year. See also: [Java 8 documentation](https://docs.oracle.com/javase/8/docs/api/java/text/SimpleDateFormat.html).

Reply:
I try to escape some words, but is not working.

Reply:
Just use a common date pattern. I use the long-form of the month in German.

Reply:
I can use the old way, with `\'`

Reply:
I don't think this will work because only a defined set of characters is allowed for the date pattern.

Reply:
I test it in other project and it is working

Reply:
Let's give a try. Maybe you found a way I am not aware of.

Reply:
Cool :heart_decoration: It works. Thanks for mentioning that!